### DISCUSSION TITLE
Do Winget manifests autoupdate or is the process manual?

### DISCUSSION THREAD

**Discussion by pratikpc** [microsoft/winget-cli#863] [Q&A] [answered]
So for example, let us say an obscure package that I today need which results in me adding support for it via Winget.

Now this package is fairly good when it comes to releases and just like Winget they use GitHub Releases and mark their latest binaries as latest.

In fact the package is so great they even name their files in a simple manner with [https://github.com]/\<owner\>/\<repo\>/releases/\<version\>/app.exe?

So do I need to keep an eye whenever the package updates and then update the manifest?

Or does the manifest support auto-updation?

Kinda like Scoop

**Comment by denelon** [comment, marked as answer]
We have a feature in the backlog to look at GitHub releases as a way to trigger an automatic update, but it's not implemented. In this case, someone would need to maintain the manifests.

https://github.com/microsoft/winget-pkgs/issues/500
https://github.com/microsoft/winget-pkgs/issues/1515

I believe @isaacrlevin has a working example of a GitHub action to do such a thing.

**Comment by isaacrlevin** [comment]
I have a process that worked with the old schema version, I still need to update to adapt to the new schema. But here is the part of my GitHub Action that updates the package in winget.

https://github.com/isaacrlevin/presencelight/blob/768514b28ca16c4407441391736a42eb3c9551c7/.github/workflows/Desktop_Build.yml#L506
